account_multicurrency_revaluation_rate_type/model/res_config.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class AccountConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    revaluation_rate_type = fields.Selection(
+        related='company_id.revaluation_rate_type',
+        string='Rate type *',
```

Comment:
```suggestion
        string='Revaluation rate type *',
```
string arg to update.